catkit2/services/hamamatsu_camera/hamamatsu_camera.py

```diff
@@ -22,6 +23,41 @@
     raise
 
 
+def _create_property(hamamatsu_property_name, read_only=False, stopped_acquisition=True):
+    def getter(self):
+        with self.mutex:
+            if hamamatsu_property_name == 'EXPOSURETIME':
+                return self.cam.prop_getvalue(getattr(dcam.DCAM_IDPROP, hamamatsu_property_name)) * 1e6
+            elif hamamatsu_property_name in ['IMAGE_WIDTH', 'IMAGE_HEIGHT', 'SUBARRAYHSIZE', 'SUBARRAYVSIZE', 'SUBARRAYVPOS', 'SUBARRAYHPOS']:
+                return int(self.cam.prop_getvalue(getattr(dcam.DCAM_IDPROP, hamamatsu_property_name)))
+            else:
+                return self.cam.prop_getvalue(getattr(dcam.DCAM_IDPROP, hamamatsu_property_name))
+
+    if read_only:
+        setter = None
+    else:
+        def setter(self, value):
+            was_running = self.is_acquiring.get()[0] > 0
+
+            if was_running and stopped_acquisition:
+                self.end_acquisition()
+
+                while self.is_acquiring.get()[0]:
+                    time.sleep(0.001)
+
+            with self.mutex:
+                if hamamatsu_property_name == 'EXPOSURETIME':
+                    self.cam.prop_setvalue(getattr(dcam.DCAM_IDPROP, hamamatsu_property_name), value / 1e6)
+                elif hamamatsu_property_name in ['IMAGE_WIDTH', 'IMAGE_HEIGHT', 'SUBARRAYHSIZE', 'SUBARRAYVSIZE', 'SUBARRAYVPOS', 'SUBARRAYHPOS']:
+                    return int(self.cam.prop_setvalue(getattr(dcam.DCAM_IDPROP, hamamatsu_property_name), value))
```

Comment:
This is a setter, not a getter, so it shouldn't return anything (as it didn't before).